fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -1890,7 +1890,7 @@ private static void validateHTML(final String path) throws IOException, SAXExcep
             htmlParser.setDoctypeExpectation(NO_DOCTYPE_ERRORS);
             htmlParser.setErrorHandler(new HTMLErrorHandler());
             htmlParser.setContentHandler(new TreeBuilder());
-            try (final InputStream htmlStream = new ByteArrayInputStream(content.getBytes())) {
+            try (final InputStream htmlStream = new ByteArrayInputStream(content.getBytes("utf8"))) {
```

Comment:
I would recommend using `java.nio.charset.StandardCharsets.UTF_8` instead of a `String` here.